src/ploomber/cli/examples.py

```diff
@@ -259,11 +261,24 @@ def list(self):
         tw.write('Example: ploomber examples -n templates/ml-basic -o ml\n\n')
 
     def download(self, name, output):
+        with open(self.examples / '_index.csv',
+                  newline='',
+                  encoding='utf-8-sig') as f:
+            rows = list(csv.DictReader(f))
         selected = self.path_to(name)
-
+        categories = []
+        for row in rows:
+            category = row.pop('name')
+            del row['idx']
+            categories.append(category)
         if not selected.exists():
+            closest_match = _suggest_command(name, categories)
+            # when suggested command returns None, disable did you mean feature
+            did_you_mean_message = f'Did you mean "{closest_match}"?\n' \
+                if closest_match is not None else ''
             raise BaseException(
-                f'There is no example named {name!r}.\n'
+                f'There is no example named {name!r}. '
+                f'{did_you_mean_message}'
```

Comment:
we need a conditional here. if there's not suggestion, we should print

> there is no message named... list examples... update...

but if there is a suggestion, then the message should only be

> there is no message named... did you mean ...?

Reply:
I handled the conditional in this way: 
```python
            did_you_mean_message = f'Did you mean "{closest_match}"?\n' \
                if closest_match is not None else ''
```
https://github.com/qixuan27/ploomber/blob/9751d3fa2874c462591f8e473fbc883a645fe85e/src/ploomber/cli/examples.py#L277-L278

Does that look OK?

Reply:
the problem with the current logic is that If there is a suggestion the error message is too verbose. it looks like this:

```
there is not example named ...
list examples...
update local copy...
get ml example...
```

but it should be more concise, and just:

```
there is not example named ...
```

when there is *no* suggestion, the error is fine. no need to change anything:

```
list examples...
update local copy...
get ml example...
```